Fody/ConfigFileFinder.cs

```diff
@@ -18,23 +24,117 @@ public static List<string> FindWeaverConfigs(string solutionDirectoryPath, strin
         var projectConfigFilePath = Path.Combine(projectDirectory, "FodyWeavers.xml");
         if (!File.Exists(projectConfigFilePath))
         {
-            logger.LogDebug($@"Could not file a FodyWeavers.xml at the project level ({projectConfigFilePath}). Some project types do not support using NuGet to add content files e.g. netstandard projects. In these cases it is necessary to manually add a FodyWeavers.xml to the project. Example content:
+            if (!files.Any() && (wellKnownWeaverFiles != null))
+            {
+                GenerateDefault(projectConfigFilePath, wellKnownWeaverFiles);
+
+                logger.LogError($@"Could not file a FodyWeavers.xml at the project level ({projectConfigFilePath}). A default file has been created. Please review the file and add it to your project.");
+            }
+            else
+            {
+                logger.LogDebug($@"Could not file a FodyWeavers.xml at the project level ({projectConfigFilePath}). Some project types do not support using NuGet to add content files e.g. netstandard projects. In these cases it is necessary to manually add a FodyWeavers.xml to the project. Example content:
```

Comment:
Same as above, but we probably also should log this as an error.

```suggestion
                logger.LogError($@"Could not find a FodyWeavers.xml file at the project level ({projectConfigFilePath}). Some project types do not support using NuGet to add content files e.g. netstandard projects. In these cases it is necessary to manually add a FodyWeavers.xml to the project. Example content:
```